lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
I would expect it to be in the initilizer, together with :https://github.com/Coursemology/coursemology2/blob/master/config/initializers/polyglot.rb

But if we do that way, the gem side need to have some changes, What do you think ?